### ISSUE TITLE
Timeout when sending monitor events crashes app

### ISSUE THREAD

**Issue by davidfokkema** [HiSPARC/pysparc#26]
This should not happen:

``` python
Traceback (most recent call last):
  File "/opt/pysparc/scripts/test_hv.py", line 140, in <module>
    app.run()
  File "/opt/pysparc/scripts/test_hv.py", line 104, in run
    self.monitor.send_uptime()
  File "/opt/pysparc/pysparc/monitor.py", line 49, in send_uptime
    self._send_status_for_service('Uptime Pi', OK, uptime)
  File "/opt/pysparc/pysparc/monitor.py", line 122, in _send_status_for_service
    requests.get(MONITOR_URL, params=payload, timeout=2)
  File "/usr/local/lib/python2.7/dist-packages/requests/api.py", line 69, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/api.py", line 50, in req
    response = session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 465,
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python2.7/dist-packages/requests/sessions.py", line 605,
    r.content
  File "/usr/local/lib/python2.7/dist-packages/requests/models.py", line 750, in
    self._content = bytes().join(self.iter_content(CONTENT_CHUNK_SIZE)) or bytes
  File "/usr/local/lib/python2.7/dist-packages/requests/models.py", line 673, in
    for chunk in self.raw.stream(chunk_size, decode_content=True):
  File "/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/respons
    for line in self.read_chunked(amt, decode_content=decode_content):
  File "/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/respons
    self._update_chunk_length()
  File "/usr/local/lib/python2.7/dist-packages/requests/packages/urllib3/respons
    line = self._fp.fp.readline()
  File "/usr/lib/python2.7/socket.py", line 447, in readline
    data = self._sock.recv(self._rbufsize)
socket.timeout: timed out
```

We should only log a warning and continue trying.

**Comment by davidfokkema** [comment]
Oh, it *doesn't* crash the app, just the main thread, I think. The app keeps running and is not restarted by supervisor.

**Comment by davidfokkema** [comment]
Hopefully fixed by 8704912.
